Gemfile.lock

```diff
@@ -431,6 +435,14 @@ GEM
       simplecov-html (~> 0.10.0)
     simplecov-html (0.10.0)
     single_cov (0.2.2)
+    slack-ruby-client (0.7.4)
```

Comment:
This library only supports sending message to a channel via webhook, which we do it via direct `Faraday` POSTs at the moment. It does not support the other slack apis, e.g., the `users_list` api that we need.